tests/unit/test_csrf_protection.py

```
@@ -0,0 +1,68 @@
+    """Enable CSRF protection for the duration of the test."""
+    original_value = app.config.get("CSRF_ENABLED")
+    app.config["CSRF_ENABLED"] = True
+    yield
+    # Restore original value
+    if original_value is not None:
+        app.config["CSRF_ENABLED"] = original_value
+    else:
+        app.config.pop("CSRF_ENABLED", None)
+
+
+def test_csrf_protection_missing_token(client, enable_csrf):
```

Ruff (0.15.2)

[warning] Unused function argument: `enable_csrf`

(ARG001)